README.md

```diff
@@ -63,20 +63,10 @@ A utility script (./tpccbenchmark) is provided for running the benchmark. The op
 ```
 
 ## Example
-The following command for example initiates a tpcc database (--create=true --load=true) and a then run a workload as described in config/workload_all.xml file. The results (latency, throughput) are summarized and written into two files: outputfile.res (aggregated) and outputfile.raw (detailed):
+The following commands initiate a tpcc database (--create=true --load=true) and a then run a workload (--execute=true) as described in config/workload_all.xml file. The results (latency, throughput) are summarized and written to stdout:
```

Comment:
Rather than "initiate" (which is something you do to a connection, or a conversation), I'd suggest "instantiate" or "initialize" a TPC-C database.

You're also deleting the part below about loading taking a long time. Did you want to retain that information somewhere, or has it gotten faster?

```suggestion
The following commands initiate a TPC-C database (`--create=true` and `--load=true`) and a then run a workload (`--execute=true`) as described in the config/workload_all.xml file. The latency and throughput results are summarized and written to stdout:
```

Reply:
@polarweasel I hear what you are saying regarding the "taking a long time" part, but that procedure necessitates compacting the database prior to backing up and restoring it for reuse.  Adding those three topics expands the scope of the README beyond the simple steps needed to quickly get started using the tool.  The original README didn't delve into any of that minutiae, so I thought it was best to excise it.  My aim was to keep the example simple.

Reply:
I appreciate your intention here. Thanks for clarifying! If you're ready to merge, merge away. :)

Reply:
(Well, except for the other changes I suggested there!)